conf/userconf/tudelft/control_panel.xml

```diff
@@ -258,76 +245,78 @@
         <arg flag="-speech"/>
       </program>
     </session>
-
     <session name="Hydrogen">
       <program name="Attitude Visualizer"/>
       <program name="HydrogenStatus"/>
       <program name="NederdroneStatus"/>
       <program name="ATC"/>
     </session>
-
     <session name="Joystick Hobbyking">
       <program name="Joystick">
         <arg flag="-ac" constant="@AIRCRAFT"/>
         <arg flag="hobbyking.xml"/>
       </program>
     </session>
-
     <session name="Joystick X3D Pro">
       <program name="Joystick">
         <arg flag="-ac" constant="@AIRCRAFT"/>
         <arg flag="extreme_3d_pro.xml"/>
       </program>
     </session>
-
     <session name="Messages and Settings">
       <program name="Messages"/>
       <program name="Settings">
         <arg flag="-ac" constant="@AIRCRAFT"/>
       </program>
     </session>
-
     <session name="PythonTools">
       <program name="IridiumDialer"/>
       <program name="PayloadForward"/>
       <program name="SVInfo"/>
       <program name="IridiumDialer"/>
     </session>
-
+    <session name="RW25">
```

Comment:
I have created a control panel for the Rotating Wing Drones (RW_control_panel). There you can find already some useful session for simulation, outdoor flight and optitrack flight. It would be nice to move these sessions there.

Reply:
I added it to the ```RW_control_panel```